src/pages/Dashboard/Dashboard.vue

```diff
@@ -207,7 +189,17 @@ export default {
 <template>
   <v-sheet color="appBackground">
     <SubPageNav>
-      <span slot="page-type">Dashboard</span>
+      <span
+        slot="page-type"
+        style="align-items: center;
+      display: flex;
+      flex-direction: column;
+      font-size: 0.75rem;
+      height: 100%;
+      justify-content: center;
+      text-align: center;"
+        ><v-icon>view_quilt</v-icon>DASH</span
```

Comment:
Is this meant to be dashboard?

Reply:
<img width="503" alt="Screen Shot 2020-11-24 at 11 11 51 AM" src="https://user-images.githubusercontent.com/40272060/100120419-eab0ab80-2e45-11eb-9a9e-b85b63ecd85e.png">

Reply:
Nicholas and I discussed this yesterday, since all the other pages have short names (flow, task, etc) we figured Dash was ok to make the spacing more consistent :)

Reply:
I think we still need to work on that spacing/design though, it's not quite there yet

Reply:
The updated design was merged with #463